native/src/routes/search/components/NothingFoundFeedbackBox.js

```diff
@@ -85,4 +97,10 @@ const NothingFoundFeedbackBox = ({ query, t, theme, sendFeedback }: PropsType) =
   }
 }
 
-export default NothingFoundFeedbackBox
+export default (props: PropsType) => (
+  <Container>
+    <FeedbackBox>
+      <NothingFoundFeedbackBox {...props} />
+    </FeedbackBox>
+  </Container>
+)
```

Comment:
This should be added the the NothingFoundFeedbackBox component. Else we have here an anonymous component.